Gopkg.toml

```diff
@@ -74,7 +74,9 @@
 
 [[constraint]]
   name = "gopkg.in/src-d/lookout-sdk.v0"
-  version = "0.0.4"
+  # there is no release of sdk with new ID for review events yet
+  # version = "0.0.4"
+  revision = "e06a088af4fe40561700c057c5dacd46e23b2abe"
```

Comment:
@carlosms @dpordomingo would it make sense to release SDK first, as we discussed couple of days ago, and then put 0.0.5 here?

Reply:
makes perfect sense, yes.

Reply:
https://github.com/src-d/lookout-sdk/releases/tag/v0.1.0 🚀